options/options.html

```diff
@@ -36,7 +43,10 @@
         When a URL of a pinned tab matches one of these case insensitive <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions">regular expressions</a>, its links will not be changed. Use this to ignore sites that already behave like pinned apps. Example: <code>twitter\.com</code>
       </p>
       <div class="info"><span></span>&nbsp;</div>
-      <textarea class="optionValue" rows="10" cols="50"></textarea>
+      <textarea id="option" rows="10" cols="50" class="optionValue"></textarea>
+
+      <p class="title"><b>Behaviour</b></p>
+      <label><input type="checkbox" id="optionStickDomain" class="optionValue" value="checked"> Open internal links in the pinned tab</label>

```

Comment:
What about making this more explicit? _Open same-origin links in the pinned tab._

Reply:
I'm not a native English speaker and struggled with the wording. 👍